pkg/archive/archive_test.go

```diff
@@ -1244,6 +1245,8 @@ func TestXGlobalNoParent(t *testing.T) {
 // header entries are created recursively with the default mode (permissions) stored in ImpliedDirectoryMode. This test
 // also verifies that the permissions of explicit directories are respected.
 func TestImpliedDirectoryPermissions(t *testing.T) {
+	testutil.SkipWhenUnprivileged(t)
+
```

Comment:
nit: it'd be reasonable to group the skips together (i.e., remove the blank line here)

Reply:
Included in next changeset.